pinot-core/src/test/java/org/apache/pinot/queries/InnerSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -175,26 +178,23 @@ public void testVeryLargeAggregationGroupBy() {
   @Test
   public void testSingleColumnDistinct() {
```

Comment:
Can we add another 2 similar tests for single and multi column distinct on dictionary encoded columns. Here both the tests are on raw columns (and so we use DistinctTable)

Reply:
`DistinctQueriesTest` has all the scenarios covered